Firestore/Example/Tests/Util/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(XCTest REQUIRED)
+
+objc_framework(
+  firebase_firestore_tests_util
+  HEADERS
+    FSTEventAccumulator.h
+    # XCTestCase+Await.h
```

Comment:
Note the hack here: I moved `XCTestCase+Await` into `FSTIntegrationTestCase` to avoid category..I could not figure out how to make that work.

Reply:
The way to get this to work is to modify the linker command line to include the `-ObjC` flag. Use `target_link_options` like so:

```cmake
target_link_options(
  firebase_firestore_tests_util
  PUBLIC -ObjC
)
```

This might not work though, in which case this may need to be set as a private option on executables that link all this stuff together.

Reply:
I've restored XCTestCase+Await into a separate file, but had to work around this by explicitly linking. If you agree this is OK, let's defer further work on this.